examples/gallery/embellishments/map_roses.py

```diff
@@ -0,0 +1,149 @@
+r"""
+Directional map roses
+=========
+
+The ``rose`` parameter of the :meth:`pygmt.Figure.basemap` and
+:meth:`pygmt.Figure.coast` methods is used to add a directional map 
+rose to a map. This example shows how such a map rose can be customized:
+
+ - position: **g**\|\ **j**\|\ **J**\|\ **n**\|\ **x**. Set the position
+   of the reference point. Choose from
+
+   - **g**: Give map coordinates as *longitude*\/\ *latitude*.
+   - **j**\|\ **J**: Specify a two-character (order independent) code.
+     Choose from vertical **T**\(op), **M**\(iddle), or **B**\(ottom) and
+     horizontal **L**\(eft), **C**\(entre), or **R**\(ight). Lower /
+     uppercase **j** / **J** mean inside / outside of the map bounding
+     box.
+   - **n**: Give normalized bounding box coordinates as *nx*\/\ *ny*.
+   - **x**: Give plot coordinates as *x*\/\ *y*.
+
+- width: **+w**: Set the width of the rose in plot coordinates (append
+  **i** (inch), **cm** (centimeters), or **p** (points).
+
+- fanciness: **+f**[level]: Get a “fancy” rose, and optionally specify the
+  level of fanciness. Level 1 draws the two principal E-W, N-S orientations,
+  2 adds the two intermediate NW-SE and NE-SW orientations, while 3 adds
+  the eight minor orientations WNW-ESE, NNW-SSE, NNE-SSW, and ENE-WSW
+  [default is 1].
```

Comment:
What's about having a list here?

```suggestion
- fanciness: **+f**[level]: Get a "fancy" rose, and optionally specify the
  level of fanciness. 
  - ``"1"``: Draws the two principal E-W, N-S orientations`.
  - ``"2"``: Adds the two intermediate NW-SE and NE-SW orientations.
  - ``"3"``: Adds the four minor orientations WNW-ESE, NNW-SSE, NNE-SSW, and ENE-WSW.
  [Default is ``"1"``].
```